src/components/PostDetail.js

```diff
@@ -24,6 +25,12 @@ var PostDetail = React.createClass({
         <Text style={styles.caption}>{post.image.caption}</Text>
       );
     }
+    var byline;
+    if (post.authors){
+      byline = (
+        <Text style={styles.byline}>By {post.authors.toString(" ")}</Text>
```

Comment:
I think [`join()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) would be more appropriate than `toString`